examples/rustapi_module/tests/test_subclassing.py

```diff
@@ -1,9 +1,13 @@
+import platform
+
 from rustapi_module.subclassing import Subclassable
 
+PYPY = platform.python_implementation() == 'PyPy'
 
 class SomeSubClass(Subclassable):
     pass
 
 
-a = SomeSubClass()
-_b = str(a) + repr(a)
+if not PYPY:
```

Comment:
This should be moved into a test function, like:

```python
def test_subclass_repr():
    a = SomeSubclass()
    _b = str(a) + repr(a)
```

Or maybe into several tests, since it seems like it's testing more than one thing here.

When it's a separate test, it should be marked `xfail` on PyPy unless it's causing a segfault, in which case it should be skipped conditionally on PyPy.

Reply:
Putting this in a test function is what I've originally done (being a good boy-scout), but this actually causes cpython to segfault, not pypy https://github.com/PyO3/pyo3/issues/407.
So I suggest handling this separately.